plugin/wakatime.vim

```diff
@@ -38,10 +38,15 @@ let s:VERSION = '9.0.1'
     endif
 
     " Script Globals
-    let s:home = expand("$WAKATIME_HOME")
-    if s:home == '$WAKATIME_HOME'
+    let s:home = ''
+    if exists('$WAKATIME_HOME')
+        let s:home = expand('$WAKATIME_HOME')
+    else
         let s:home = expand("$HOME")
     endif
+    if has('win32')
```

Comment:
Would it hurt to always replace `\` with `/`, even when not on Windows?

Reply:
Tested on Windows 11, Msys2 and WSL no problem.